ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverManagerImpl.java

```diff
@@ -15,11 +19,18 @@
     Session session;
 
     @Override
-    public Cluster registerCluster(String clName, String contactPoint, int port) {
+    public Cluster registerCluster(String clName, String contactPoint, int connections, int port) {
+    
+        PoolingOptions poolingOpts = new PoolingOptions()
+                                     .setConnectionsPerHost(HostDistance.LOCAL, connections, connections)
```

Comment:
Why are you repeating `connections` twice as an input argument?

Reply:
See the method *.setConnectionsPerHost* here:

https://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/core/PoolingOptions.html#setConnectionsPerHost-com.datastax.driver.core.HostDistance-int-int-